go.mod

```diff
@@ -256,6 +256,7 @@ require (
 )
 
 replace (
+	github.com/argoproj/gitops-engine => github.com/gdsoumya/gitops-engine v0.0.0-20230213130857-26f153a15602
```

Comment:
Will need to merge https://github.com/argoproj/gitops-engine/pull/507 and update the go.mod before merging this PR

Reply:
https://github.com/argoproj/gitops-engine/pull/507 is merged. Please update go.mod

Reply:
Thanks! updated.